package.json

```diff
@@ -26,85 +26,27 @@
         "default": "./esm/index.mjs"
       },
       "module": {
-        "types": "./index.d.ts",
+        "types": "./esm/index.d.ts",
         "default": "./esm/index.js"
       },
       "default": {
         "types": "./index.d.ts",
         "default": "./index.js"
       }
     },
-    "./vanilla": {
-      "types": "./vanilla.d.ts",
-      "import": {
-        "types": "./esm/vanilla.d.mts",
-        "default": "./esm/vanilla.mjs"
-      },
-      "module": "./esm/vanilla.js",
-      "default": "./vanilla.js"
-    },
-    "./middleware": {
-      "types": "./middleware.d.ts",
-      "import": {
-        "types": "./esm/middleware.d.mts",
-        "default": "./esm/middleware.mjs"
-      },
-      "module": "./esm/middleware.js",
-      "default": "./middleware.js"
-    },
-    "./middleware/immer": {
-      "types": "./middleware/immer.d.ts",
-      "import": {
-        "types": "./esm/middleware/immer.d.mts",
-        "default": "./esm/middleware/immer.mjs"
-      },
-      "module": "./esm/middleware/immer.js",
-      "default": "./middleware/immer.js"
-    },
-    "./shallow": {
-      "types": "./shallow.d.ts",
-      "import": {
-        "types": "./esm/shallow.d.mts",
-        "default": "./esm/shallow.mjs"
-      },
-      "module": "./esm/shallow.js",
-      "default": "./shallow.js"
-    },
-    "./vanilla/shallow": {
-      "types": "./vanilla/shallow.d.ts",
-      "import": {
-        "types": "./esm/vanilla/shallow.d.mts",
-        "default": "./esm/vanilla/shallow.mjs"
-      },
-      "module": "./esm/vanilla/shallow.js",
-      "default": "./vanilla/shallow.js"
-    },
-    "./react/shallow": {
-      "types": "./react/shallow.d.ts",
-      "import": {
-        "types": "./esm/react/shallow.d.mts",
-        "default": "./esm/react/shallow.mjs"
-      },
-      "module": "./esm/react/shallow.js",
-      "default": "./react/shallow.js"
-    },
-    "./traditional": {
-      "types": "./traditional.d.ts",
+    "./*": {
```

Comment:
This requires newer version of Node, and we will be doing this change in v5.

Reply:
@dbritto-dev We are waiting for the revert.

Reply:
@dai-shi done

Reply:
Ooops, sorry. I don't mean to revert #2157 fully.
Can you try what you think should work, and just avoid using `"./*"`.

Reply:
In that we case we should apply the same change for each inner package until we can use the `./*`

Reply:
Yes, that what I meant.

Reply:
Sure, let me do it